node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -88,3 +90,87 @@ func TestTokenListEmptyCoinGeckoId(t *testing.T) {
 		assert.Greater(t, len(tokenConfigEntry.coinGeckoId), 0)
 	}
 }
+
+// If not true, then there exists some chain that Wormhole governs yet has no governed tokens. That means assets
```

Comment:
Tokens which are governed affect all chains they are transferred to/from. Why should it be necessary that a chains native assets are governed?

Reply:
There was a time when we had governed chains that did not have governed tokens. See lines 51 of this file. I guess that line is obsolete now, but there was a reason that line was added.

Reply:
Ah thanks Bruce, I missed that somehow. I think because I stated with an approach that iterated over all the vaa ChainIDs so at first it looked like there _were_ some that were missing.
I'll take some time to review my test and see if it adds anything new. If not I'll close the PR, otherwise I might rework the existing test rather than add a mostly-redundant one.

Reply:
Actually, how is this test different from `TestTokenListChainTokensPresent`?

Reply:
It's not - I didn't see the other test 😅 
I've changed the PR so that it updates the existing test to be a little more verbose and commented. It's fine as-is but since I already wrote the comments, might as well add them here.
I deleted my new test and have (hopefully) combined the best of both worlds.